Gruntfile.coffee

```diff
@@ -298,9 +298,9 @@ module.exports = (grunt) ->
 				cwd: "site/img"
 				src: "**/*.*"
 				dest: "dist/img"
-			site_assets_dist:
+			site_assets_min:
 				expand: true
-				cwd: "dist/unmin"
+				cwd: "site/pages"
```

Comment:
look very similar like "site_assets". Consider to include multi "dest" if supported or put both one to each other